Easier spark shell "how-to" guide

maasg opened the pull request:
Attempts to make the how-to guide on using the Spark-Cassandra Connector with the Spark Shell more accessible by:
 - moving detail information out of the main path, 
 - adding a small paragraph on why this is needed
 - some style refactoring

@RussellSpitzer what do you think?  (btw, I didn't know if I needed to track this into JIRA, and there's an umbrella story for that as well. https://datastax-oss.atlassian.net/browse/SPARKC-41  Let me know if I need to take action there in case this proposal is deemed of some use)

maasg commented on doc/13_1_setup_spark_shell.md:
```diff
@@ -0,0 +1,32 @@
+
+The easiest way to get quickly started with Cassandra is to follow the instructions provided by 
+[Datastax](http://docs.datastax.com/en/cassandra/2.1/cassandra/install/install_cassandraTOC.html)
+
```
@RussellSpitzer  should we mention a docker alternative? That's a quite low-threshold way to quickly build up a playground. Al Tobey maintains one image: https://github.com/tobert/cassandra-docker

RussellSpitzer replied:
Sounds good to me, Don't tell Al but I haven't tried his image yet ;)

RussellSpitzer commented:
This looks great to me, just let me know if you want to add Docker instructions or a link if you like otherwise I'll just merge.

maasg commented:
@RussellSpitzer I've added the docker section. I found it fitted better on the same page as the spark-shell guide.  WDYT? 

Also validated the spark-shell guide and adjusted the version accordingly. Note that it now builds a tag instead of a branch. I found that things don't work with some minor version combinations. Eg. Spark 1.2.2 + Connector 1.2.1 = :skull:

pkolaczk commented:
+1

pkolaczk commented:
@RussellSpitzer if you find it good as well, merge it

RussellSpitzer commented:
+1 Merging, thanks @maasg !
